src/Dmishh/Bundle/SettingsBundle/Manager/SettingsManagerInterface.php

```diff
@@ -20,52 +20,52 @@
     const SCOPE_USER   = 'user';
 
     /**
-     * Returns setting value by its name
+     * Returns setting value by its name.
      *
-     * @param string $name
+     * @param string            $name
      * @param SettingOwner|null $owner
      *
-*@return mixed
+     * @return mixed
      */
-    function get($name, SettingOwner $owner = null);
+    public function get($name, SettingOwner $owner = null);
```

Comment:
That was what I ment with my question earlier: 

> Should the default parameter be forced in the SettingsManagerInterface?

I think it would be a good idea.